cmd/mouseRecon/mouseRecon.go

```diff
@@ -0,0 +1,141 @@
+		} else {
+			mraNode.Fasta.Seq = append(mraNode.Fasta.Seq, dna.Gap)
+		}
+	}
+	fasta.Write(outFile, []fasta.Fasta{*mouseNode.Fasta, *ratNode.Fasta, *hamsterNode.Fasta, *squirrelNode.Fasta, *mraNode.Fasta})
+
+}
+
+func mraIsPresent(mouse, rat, hamster, squirrel dna.Base) bool {
+	if baseIsPresent(mouse) && baseIsPresent(rat) {
+		return true
+	}
+	if (baseIsPresent(mouse) || baseIsPresent(rat)) && (baseIsPresent(hamster) || baseIsPresent(squirrel)) {
+		return true
+	}
+	return false
+}
+
+func baseIsPresent(b dna.Base) bool {
+	if dna.DefineBase(b) || b == dna.N {
+		return true
+	}
+	return false
+}
+
+func reconMraBase(root, mouseNode, nodeToRecon *expandedTree.ETree, position int, probThreshold float64, nonBiasProbThreshold float64) {
+	var likelihoods []float64
+	var nextBase dna.Base
+	reconstruct.SetState(root, position)
+	likelihoods = reconstruct.FixFc(root, nodeToRecon)
+	nextBase = likelihoodToBaseBias(likelihoods, mouseNode.Fasta.Seq[position], probThreshold, nonBiasProbThreshold)
+	nodeToRecon.Fasta.Seq = append(nodeToRecon.Fasta.Seq, nextBase)
+}
+
+func likelihoodToBaseBias(likes []float64, biasBase dna.Base, probThreshold float64, nonBiasProbThreshold float64) dna.Base {
+	var total, nonBiasTotal, bestProb float64
+	var i int
+	var answer dna.Base
+
+	if biasBase == dna.Gap {
+		answer = dna.N
+	} else {
+		answer = biasBase
+	}
+
+	for i = range likes {
+		total += likes[i]
+		if i != int(biasBase) {
+			nonBiasTotal += likes[i]
+		}
+	}
+
+	for i = range likes {
+		if likes[i]/total >= probThreshold && likes[i] > bestProb && nonBiasTotal/total >= nonBiasProbThreshold {
+			bestProb = likes[i]
+			answer = dna.Base(i)
+		}
+	}
+	return answer
+}
+
+func usage() {
+	fmt.Print(
+		"mouseRecon - Returns maximum likelihood ancestral sequences from a Mouse-Rat-ChineseHamster-Squirrel multiple alignment in multiFa format.\n" +
+			"Usage:\n" +
+			"mouseRecon input.fa output.fa\n" +
+			"options:\n")
+	flag.PrintDefaults()
+}
+
+func main() {
+	var expectedNumArgs int = 2
+	var tree *string = flag.String("mleTree", "", "Filename for newick tree with branch lengths. Must hav ethe anticipated assembly names and the desired ancestral node labeled.")
+	var probThreshold *float64 = flag.Float64("probThreshold", 0.0, "The minimum probability assigned to a base required for it to be called.")
+	var nonBiasProbThreshold *float64 = flag.Float64("nonBiasProbThreshold", 0.0, "The summation of all base probabilities (other than the biased species) must pass this threshold for a non-biased species base to be considered in the ancestral state.")
+	//var mleMraUnbiased *bool = flag.Bool("mleMraUnbiased", false, "Calculates the unbiased mouse-rat ancestor estimation. Currently unsupported (future feature).")
+	var mleMraMouseBiased *bool = flag.Bool("mleMraMouseBiased", true, "Calculates the mouse-biased mouse-rat ancestor estimation (default behavior, only behavior currently supported).")
```

Comment:
flag option but not used in the code? If always biased for now, maybe comment out for now and describe in the usage?